docs/en/Running-Alluxio-On-Docker.md

```diff
@@ -44,7 +44,31 @@ $ docker build -t alluxio .
 By default, this will build an image for the latest released version of Alluxio. To build
 from a local Alluxio tarball instead, you can use `--build-arg`
 ```bash
-docker build -t alluxio --build-arg ALLUXIO_TARBALL=alluxio-snapshot.tar.gz .
+$ docker build -t alluxio --build-arg ALLUXIO_TARBALL=alluxio-snapshot.tar.gz .
+```
+
+### Set up ramdisk to enable short-circuit reads
+
+When the Alluxio client runs local to an Alluxio worker, a shared ramdisk should be set
+up so that short-circuit reads can be used to read data at memory speed instead of
+network speed.
+
+```bash
+$ sudo mkdir /mnt/ramdisk && sudo mount -t ramfs -o size=10G tmpfs /mnt/ramdisk
+```
+
+Alternately, you can go to `conf/alluxio-site.properties` on the host system, set
```

Comment:
yea, on further thought I've removed it since it's weird to expect Alluxio to be installed on the host machine in a Docker setting.